src/graph/graph.c

```diff
@@ -223,6 +220,32 @@ void _Graph_GetEdgesConnectingNodes(const Graph *g, NodeID src, NodeID dest, int
 	}
 }
 
+// Locates edges connecting src to destination.
+void _Graph_GetEdgesConnectingNodes
+(
+	const Graph *g,
+	NodeID src,
+	NodeID dest,
+	int r,
+	Edge **edges
+) {
+	ASSERT(g);
+	ASSERT(r    != GRAPH_NO_RELATION);
+	ASSERT(r    < Graph_RelationTypeCount(g));
+	ASSERT(src  < Graph_RequiredMatrixDim(g));
+	ASSERT(dest < Graph_RequiredMatrixDim(g));
+
+	// relation map, maps (src, dest, r) to edge IDs.
+	EdgeID      id    =  INVALID_ENTITY_ID;
```

Comment:
Need to check if INVALID_ENTITY_ID value is correct after MSB change

Reply:
This is just a default value which will be overwritten by `GrB_Matrix_extractElement_UINT64`